shared/src/main/kotlin/com/egm/stellio/shared/util/ApiUtils.kt

```diff
@@ -81,10 +82,14 @@ fun hasValueInOptionsParam(options: Optional<String>, optionValue: OptionsParamV
         .filter { it.any { option -> option == optionValue.value } }
         .isPresent
 
-fun parseAndExpandAttrsParameter(attrsParam: String?, contextLink: String): Set<String> =
-    attrsParam
+fun parseRequestParameterAsSet(requestParam: String?): Set<String> =
```

Comment:
the `AsSet` information in the function name is in the signature of the function

Reply:
and it won't work with any request parameter ... how to make explicit what you're expecting?